pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
 
     <artifactId>liquibase-aws-license-service</artifactId>
-    <version>1.0.2-SNAPSHOT</version>
+    <version>1.0.3-SNAPSHOT</version>
```

Comment:
Why'd you change the version here?

Reply:
The Draft version should  be the next version and this is the change @filipelautert recommended we do to attach the artifacts with `v1.0.3` instead of `v1.0.2` in the draft release. 
@filipelautert

Reply:
Current released version is 1.0.2 , so I asked her to upgrade it here to prevent mistakes.